src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,35 @@
 // your components will all go in this `component` directory.
 // feel free to change this component.js into TodoList.js
+
+import React from 'react';
+import './Todo.css';
+import Task from './Task';
+import completedTaskFunction from './TodoForm';
```

Comment:
`completedTaskFunction` does not exist in TodoForm as a regular function. It's inside the TodoForm as a method, so you'll need to pass it down as a prop.